modules/ossm-cert-manager-installation.adoc

```diff
@@ -12,15 +12,7 @@ You can install the `cert-manager` tool to manage the lifecycle of TLS certifica
 
 . Create the root cluster issuer:
 +
-[source,terminal]
-----
-$ oc apply -f cluster-issuer.yaml
-----
-+
```

Comment:
Moved these steps, as the cluster-issuer.yaml file doesn't exist until the following step.